docs/running-workflows/launching-workflows/index.md

```diff
@@ -135,22 +135,13 @@ The standard set of badges look as follows:
 
 ![Launch with Snakemake REANA at CERN](https://www.reana.io/static/img/badges/launch-with-snakemake-on-reana-at-cern.svg)
 
-If you would like to generate your own custom badge, you can use external services such as [shields.io](https://shields.io) or [badgen.net](https://badgen.net).
+To streamline the badge creation process, you can use the tool accessible on REANA at the
```

Comment:
Maybe let's explain that this is instance-dependent?
Something like: ... accessible on your REANA instance at `/launcher-badge`. If you have a CERN account, you can make use of launch badge creator of the available REANA@CERN instance.

Reply:
Okay, I agree! Should we do something similar for the other examples in the rest of the page? Even though maybe for them it's probably better to leave `reana.cern.ch`...